modulefiles/ufs_hera.intel_debug.lua

```diff
@@ -12,7 +12,13 @@ load("sutils")
 cmake_ver=os.getenv("cmake_ver") or "3.20.1"
 load(pathJoin("cmake", cmake_ver))
 
-prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+intel_ver=os.getenv("intel_ver") or "2022.1.2"
+load(pathJoin("intel", intel_ver))
+
+impi_ver=os.getenv("impi_ver") or "2022.1.2"
+load(pathJoin("impi", impi_ver))
+
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/modulefiles/stack)
 
 hpc_ver=os.getenv("hpc_ver") or "1.1.0"
 load(pathJoin("hpc", hpc_ver))
```

Comment:
@jkbk2004 - Fixed the double quote, and also the hpc_ver! (1.1.0 vs. a newer 1.2.0)

Reply:
@jkbk2004 - please note that the PR is open (closed by accident!)